From 9e8c1b56bb5c999d27a6b0e26efc3000b05355ac Mon Sep 17 00:00:00 2001
From: Ken Ma <make@marvell.com>
Date: Wed, 26 Jul 2017 10:53:02 +0800
Subject: [PATCH 1099/1345] Revert "dts: a37xx: change CPU decode window
 mapping for PCIe"

commit  1f4c1ec3183fc72cca45a9cf616782f9f9436ecb from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

This reverts 'commit 303e919a6411
("dts: a37xx: change CPU decode window mapping for PCIe")'.

The patch solves the backward compability issue prior to
Armada3700 17.06 BSP release. The correlated patches of
other BSP components (ATF & U-boot) should be applied as
well.

Regarding 4GB memory support, please refer to the release
notes.

Change-Id: I88ae3e7e41aee0dec672fe3f4a8c2af0e4140522
Signed-off-by: Ken Ma <make@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/42268
Reviewed-by: Wilson Ding <dingwei@marvell.com>
Tested-by: Wilson Ding <dingwei@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-37xx.dtsi |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-37xx.dtsi b/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
index e848d16..7714bce0 100644
--- a/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
@@ -544,8 +544,8 @@
 			msi-parent = <&pcie0>;
 			msi-controller;
 			clocks = <&sb_periph_clk 13>;
-			ranges = <0x82000000 0 0xd8200000   0 0xd8200000 0 0x1000000 /* Port 0 MEM */
-				  0x81000000 0 0xd9200000   0 0xd9200000 0 0x10000>; /* Port 0 IO*/
+			ranges = <0x82000000 0 0xe8000000   0 0xe8000000 0 0x1000000 /* Port 0 MEM */
+				  0x81000000 0 0xe9000000   0 0xe9000000 0 0x10000>; /* Port 0 IO*/
 			interrupt-map-mask = <0 0 0 7>;
 			interrupt-map = <0 0 0 1 &pcie_intc 0>,
 					<0 0 0 2 &pcie_intc 1>,
-- 
1.7.9.5

